istioctl/pkg/tag/templates/validatingwebhook.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
```

Comment:
Would it be hard to move this to /manifests/charts/default/templates/ - in preparation for adding a Chart.yaml ? And also when executing the template, set the variables to match helm ( {{ .Values.tag }}, etc.

If it's too hard we can leave it for later - or just duplicate it. I have a chart with equivalent functionality in costinm/istiod/manifests/.. - I was planning to just leave it as a sample.

Alternatively, you could use the template from 'base', or add few extra {{ if }} to 
the base so it renders tags ( and helm install base --set tag=foo --set rev=bar applies just the webhooks, the rest is skipped ). This will keep things in sync.
( istioctl has code to render the chart, so may be even less code )

Reply:
Makes sense, moved the template to `manifests/charts/default` and the command now renders from there. There's also a `defaultRevision` option now on the base chart.